From 77bc9b24bd0e4d50194e9afa7187bba989a620f6 Mon Sep 17 00:00:00 2001
From: Matthieu Gallien <matthieu_gallien@yahoo.fr>
Date: Fri, 13 Dec 2019 11:32:27 +0100
Subject: [PATCH 2/3] ensure that the metadata dialog has a title and standard
 buttons

fix dialog on Windows that was without title and buttons

no regression on Plasma
---
 src/qml/MediaTrackMetadataView.qml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/qml/MediaTrackMetadataView.qml b/src/qml/MediaTrackMetadataView.qml
index b074dc4..118cea9 100644
--- a/src/qml/MediaTrackMetadataView.qml
+++ b/src/qml/MediaTrackMetadataView.qml
@@ -50,7 +50,8 @@ Window {
     }
 
     modality: Qt.NonModal
-    flags: Qt.Dialog | Qt.WindowCloseButtonHint
+    flags: Qt.Dialog | Qt.CustomizeWindowHint | Qt.WindowTitleHint
+        | Qt.WindowCloseButtonHint | Qt.WindowMinimizeButtonHint | Qt.WindowMaximizeButtonHint
 
     color: myPalette.window
 
-- 
2.24.0

